packages/vite/src/node/config.ts

```diff
@@ -813,6 +813,13 @@ export async function loadConfigFromFile(
         isTS = true
       }
     }
+
+    if (!resolvedPath) {
+      const cjsconfigFile = path.resolve(configRoot, 'vite.config.cjs')
+      if (fs.existsSync(cjsconfigFile)) {
+        resolvedPath = cjsconfigFile
```

Comment:
```suggestion
      const cjsConfigFile = path.resolve(configRoot, 'vite.config.cjs')
      if (fs.existsSync(cjsConfigFile)) {
        resolvedPath = cjsConfigFile
```

Reply:
Intentionally used the same casing as the the other config files here: `jsconfigFile`, `mjsconfigFile`, `tsconfigFile` 👀
(just fyi, doesn't matter really)